### ISSUE TITLE
We able to see the receiving address and add a label to them

### ISSUE THREAD

**Issue by jdevora** [LedgerHQ/ledger-live-desktop#1166]
- [ x] I have checked this feature was not yet requested.

I know that by default every time you press the "Receive" button you get a new address and I think it is a good idea.

BUT, there are lots of use cases for sending multiple transactions to the same address (e.g. having it registered to a service, services that impose a  lockout period when you change it...)

Right now, you have to go into the detail of each TXs to figure out the receiving address and then look the data up in a block explorer. 
An advanced option to see all the previous used incoming addresses with a link to a block explorer for each of them will make our lives much easier.

**Comment by gre** [comment]
> I know that by default every time you press the "Receive" button you get a new address and I think it is a good idea.

not exactly. It's actually only after you have receive funds to the last "Receive" address you used. if you keep receiving funds to your first address you ever had, it won't even renew an address.

> BUT, there are lots of use cases for sending multiple transactions to the same address (e.g. having it registered to a service, services that impose a lockout period when you change it...)

you can do this. you can always use the same address.

> Right now, you have to go into the detail of each TXs to figure out the receiving address and then look the data up in a block explorer.
> An advanced option to see all the previous used incoming addresses with a link to a block explorer for each of them will make our lives much easier.

this, I definitely agree

**Comment by jdevora** [comment]
I didn't realize that you only get a new address when the last one received something, but the point stands. 
There isn't an easy way to access to your previously used addresses for any purpose like
  * Reuse the address for some reason
  * Look a the transaction received
  * Add them to a full-featured portfolio tracker 
  *  ....

**Comment by jdevora** [comment]
The same is true for the outgoing addresses. 
You got an address from a service and some time after you want to see to whom you sent those crypto.
